src/api/tests/test_user_course_get.py

```diff
@@ -0,0 +1,31 @@
+from .util import Client
+
+
+
+TEST_USER = { 'email': 'test@email.com',
+              'password': '123456' }
+
+TEST_COURSE = {'name': 'ARCH-4770',
+               'cid': '-1',
+               'semester': 'SUMMER2020'}
+
```

Comment:
can you remove some of these extra new lines above and below your test data, leave two lines above and below preferably